src/clib/pio_getput_int.cpp

```diff
@@ -2465,6 +2483,24 @@ int spio_put_vars_tc(int ncid, int varid, const PIO_Offset *start, const PIO_Off
 
     if ((file->iotype == PIO_IOTYPE_ADIOS) || (file->iotype == PIO_IOTYPE_ADIOSC))
     {
+#ifdef _ADIOS2
```

Comment:
Would it be better to move the ifdef _ADIOS2 outside the if statement itself (Since this if statement and some others in this change only deal with ADIOS)?

Reply:
OK, I will make this change.